src/cubit_utils.py

```diff
@@ -0,0 +1,294 @@
+import cubit
+from pathlib import Path
+import os
+import inspect
+import subprocess
+
+
+def init_cubit(cubit_flag):
+    """Initializes Coreform Cubit with the DAGMC plugin.
+
+    Arguments:
+        cubit_flag (bool): flag to indicate whether Coreform Cubit has been
+            initialized.
+
+    Returns:
+        cubit_flag (bool): flag to indicate whether Coreform Cubit has been
+            initialized.
+    """
+    if not cubit_flag:
```

Comment:
More meaningful name here
```suggestion
    if not cubit_initialized:
```